src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/fluids/tconstruct_molten_cinderslime.json

```
@@ -0,0 +1,11 @@
+{
+  "text": "余烬黏液漂浮方向相反，爆炸也是如此。",
```

```suggestion
  "text": "余烬黏液的流动方向异于常规，爆炸也是如此。",
```
"Flow" should be referring to cinderslime flowing upwards instead of downwards, it's better to keep the original meaning of flow.